Symfony/app/config/config.yml

```diff
@@ -37,6 +37,8 @@ framework:
 twig:
     debug:            "%kernel.debug%"
     strict_variables: "%kernel.debug%"
+    globals:
+        homepage: "http://localhost:8889/"
```

Comment:
Hi John, I believe this line was reversed in a later commit. It was a temp fix for symfony's wrong redirect url.